exercises/two-bucket/src/main/java/TwoBucket.java

```diff
@@ -0,0 +1,19 @@
+
```

Comment:
As per [POLICIES](https://github.com/exercism/java/blob/master/POLICIES.md#starter-implementations), because this is higher than a difficulty of 4 we don't need the starter implementation here 🙂

Reply:
I should just delete this file?

Reply:
If you can delete it, and in its place put an empty `.keep` file :slightly_smiling_face: